### ISSUE TITLE
Test Failure Issue

### ISSUE THREAD

**Issue by sebastiansajie** [TPRU-demo/pitaxcalc-demo#74]
@martinholmer 
When running pytest the test_calculator.py/test_correct_calculator_instantiation test is giving inconsistent results. When we change the expected value to 2462.711e9 the calculated value is 2136.741e9. When we change the expected value to   2136.741e9, the calculated value flips back to 2462.711e9.

![image](https://user-images.githubusercontent.com/6294086/48345256-8b5c1600-e645-11e8-8c34-a6db07742b12.png)

![image](https://user-images.githubusercontent.com/6294086/48345646-96fc0c80-e646-11e8-99f4-d64c1459b23b.png)

**Comment by andersonfrailey** [comment]
We've figured out the issue. PR #63 adds taxes on short and long term capital gains, but very few records in the data have short and long term capital gains. So when we create a 10% subsample, it does not properly represent the larger sample and as a result produces a different weighted total for `pitax`. Because our data is already only 100 records, I've proposed commenting out the subsample test until we begin using a larger data set where a 10% subsample will be more accurate.
